src/content/updates/release-information/known-issues.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Known Issues
+tabs: ['Highlights', "Changelog", 'Known Issues']
+---
+
+## Carbon v10 identified bugs and issues
+
+Here are the most significant issues and problems that we are aware of in the Carbon v10 release. The Carbon team will be working to resolve these.
+
```

Comment:
Will we be filling in content here later?

Reply:
Yes, definitely. Once we know more known issues!